cmd/dep/init.go

```diff
@@ -68,6 +81,12 @@ func (cmd *initCommand) Run(ctx *dep.Ctx, loggers *Loggers, args []string) error
 		root = ctx.WorkingDir
 	} else {
 		root = args[0]
+		if !filepath.IsAbs(args[0]) {
```

Comment:
i checked `filepath.Clean()`, it doesn't generate absolute path and dep init fails later. 
Eg. `dep init new_test_project` would fail as filepath.clean would return `new_test_project` as the output. 

Let me know if I am missing something.